commands/hrandmember.md

```diff
@@ -0,0 +1,38 @@
+When called with just the `key` argument, return a random field from the hash value stored at `key`.
+
+If the provided `count` argument is positive, return an array of `count` **distinct fields**.
+If called with a negative `count`, the behavior changes and the command is allowed to return the **same field multiple times**. In this case, the number of returned fields is the absolute value of the specified `count`.
+
+The optional `WITHVALUES` modifier changes the reply so it includes the respective values of the randomely selected hash fields.
+
+@return
+
+@bulk-string-reply: without the additional `count` argument, the command returns a Bulk Reply with the randomly selected field, or `nil` when `key` does not exist.
+
+@array-reply: when the additional `count` argument is passed, the command returns an array of fields, or an empty array when `key` does not exist. If the `WITHVALUES` modifier is used, the reply is a list fields and their values from the hash.
```

Comment:
do we wanna mention nested arrays of RESP3? or some other time?

p.s. this one could have been broken into two lines (at the period)